components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+### Vertical Transport
+The integration in [](#z-integration-momentum) to [](#z-integration-tracers) changes the vertical velocity $w$ in m/s to a vertical mass-thickness transport $\omega=\rho w$ in kg/s/m$^2$. Here $w$ is the Latin letter and $\omega$ is the Greek letter omega.  One can think of fluid velocity $w$ as a volume transport, normalized by area, in units of length per time. Analogously, $\omega$ is mass-thickness transport, which is mass transport per unit area (kg/s/m$^2$). The variables $w$ and $\omega$ have the same sign convention of upward (positive $z$) for positive transport.
+
+### Final Layered Equations
+
+momentum:
+
+$$
+\frac{\partial {\bf u}_k}{\partial t}
++ q_k h_k {\bf u}_k^{\perp}
++ \frac{\omega_k^{top}}{\rho_k^{top}} {\bf u}_k^{top} - \frac{\omega_k^{bot}}{\rho_k^{bot}}{\bf u}_k^{bot}
+=
+- v_k \nabla_r p_k - \nabla_r \Phi_k
+- \nabla_r K_k
++ {\bf D}_k^u + {\bf F}_k^u
+$$ (layered-momentum)
+
+mass:
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla_r \cdot \left(h_k {\bf u}_k\right) + \omega_k^{top} - \omega_k^{bot}= Q^h_k
+$$ (layered-mass)
+
+tracers:
+
+$$
+\frac{\partial h_k \varphi_k}{\partial t} + \nabla_r \cdot \left(h_k {\bf u}_k \varphi_k\right)
++ \varphi_k^{top} \omega_k^{top} - \varphi_k^{bot}\omega_k^{bot}
+=  D^\varphi_k + Q^\varphi_k.
+$$ (layered-tracer)
+
+The superscripts $top$ and $bot$ mean that the layered variable is interpolated to the top and bottom layer interface, respectively. In [](layered-momentum) the non-divergent momentum advection and Coriolis term were combined and expressed in terms of the potential vorticity $q_k$,
+
+$$
+ \zeta_k {\bf u}_k^\perp + f {\bf u}_k^\perp = \frac{\zeta_k + f }{h_k}h_k{\bf u}_k^\perp \equiv q_k h_k {\bf u}^\perp_k.
+$$ (potential-vort-adv)
+
+Surface fluxes $Q^h_k$ have been added to the mass equation for precipitation, evaporation, and river runoff.  These fluxes, like mass transport $\omega$, are in units of kg/s/m$^2$.
+
+
+## 5. Discrete Equations
+
+The horizontally discretized layered equations are as follows. We have dropped the $r$ in $\nabla_r$ for conciseness, and the operator $\nabla$ from here on means within-layer.
+
+$$
+\frac{\partial u_{e,k}}{\partial t} + \left[ \frac{{\bf k} \cdot \nabla \times u_{e,k} +f_v}{[h_{i,k}]_v}\right]_e\left([h_{i,k}]_e u_{e,k}^{\perp}\right)
++ \left[ v_{i,k}^{top}\omega_{i,k}^{top} \right]_e u_{e,k}^{top} - \left[ v_{i,k+1}^{top}\omega_{i,k+1}^{top} \right]_e u_{e,k+1}^{top}
+=
+- v_{i,k} \nabla p_{i,k} - \nabla \Phi_{i,k}
+- \nabla K_{i,k} +  { \bf D}^u_{e,k} + {\bf F}^u_{e,k}
+$$ (discrete-momentum)
+
+$$
+\frac{\partial h_{i,k}}{\partial t} + \nabla \cdot \left([h_{i,k}]_e u_{e,k}\right)
++ \omega_{i,k}^{top} - \omega_{i,k+1}^{top}
+= Q^h_{i,k}
+$$ (discrete-thickness)
+
+$$
+\frac{\partial h_{i,k} \varphi_{i,k}}{\partial t} + \nabla \cdot \left(u_{e,k} [h_{i,k} \varphi_{i,k}]_e \right)
++ \varphi_{i,k}^{top} \omega_{i,k}^{top} - \varphi_{i,k+1}^{top}\omega_{i,k+1}^{top}
```

Comment:
```suggestion
+ \varphi_{i,k} \omega_{i,k}^{top} - \varphi_{i,k+1} \omega_{i,k+1}^{top}
```
Should we present @xylar and @alicebarthel's assumption that tracers are constant vertically within a layer before this equation?

Reply:
That assumption should be stated in the [layer integration section](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-doc-v1-eqns/_build/html/design/OmegaV1GoverningEqns.html#layer-integration). I will add that.

Reply:
Let's talk about this at today's meeting.  Please don't edit yet.